ppr-ui/src/utils/mhr-api-helper.ts

```diff
@@ -290,12 +290,12 @@ export async function searchMhrPDF (searchId: string): Promise<any> {
     })
 }
 
-export async function submitMhrRegistration (payloadData, queryParamData) {
+export async function submitMhrRegistration (payloadData, staffPayment) {
```

Comment:
Is this the fix for 15432?

Reply:
Yes, including the changes to the parameter formatting below.

Reply:
So by changing the params during submission it fixed the history?

Reply:
The change on line 293 is just a var name change to make the code more readable. Refactored below to not use URLSearchParams as it was returning empty, plus we already have a function to parse staff payment information

Reply:
Basically parameters were not being added to the URL for the registration POST call because URLSearchParams was returning nothing. Refactored to use mhrStaffPaymentParameters and the POST URL now contains payment info fixing the history